ms2deepscore/train_new_model/train_ms2deepscore.py

```diff
@@ -42,6 +42,7 @@ def train_ms2ds_model(
     model = SiameseSpectralModel(settings=settings)
 
     validation_loss_calculator = ValidationLossCalculator(validation_spectra,
+                                                          settings=settings,
                                                           score_bins=settings.same_prob_bins,
```

Comment:
We could remove the settings.same_prob_bins and random_seed here as well and use settings in the init.